src/parsers/js/acorn.js

```diff
@@ -24,16 +24,33 @@ export default {
   locationProps: new Set(['range', 'loc', 'start', 'end']),
 
   loadParser(callback) {
+    const parsers = {};
+    const _localReq = require;
+
     require(['acorn', 'acorn-jsx/inject'], (acorn, jsxInject) => {
```

Comment:
I think we should be able to directly require `src/loose/index.js` here and avoid all the issues with loading the built version?